src/main/java/com/verily/genomewarp/GenomeWarpSerial.java

```diff
@@ -967,7 +1019,7 @@ public static void main(String[] args) {
     VCFHeader vcfHeader = vcfReader.getFileHeader();
     ArrayList<String> vcfSampleNames = vcfHeader.getSampleNamesInOrder();
     if (vcfSampleNames.size() == 0) {
-      fail("input VCF file has no call set groups");
+      fail("input VCF file has no callset groups");
     }
     final boolean keepRefRefCalls = ARGS.keepHomozygousReferenceCalls;
 
```

Comment:
Down at line 1117, there should be a merging of contiguous regions back into single records. E.g. BED regions

chr1   10   20
chr1   20   30
chr1   30   35

should be merged into a single record

chr1   10   35

So that the output BED doesn't have way different numbers of records than the input BED.